app/models/user.rb

```diff
@@ -13,7 +13,7 @@ class User < ApplicationRecord
   scope :pending_activation, -> { where(auth_subject_id: nil, deactivated_at: nil) }
 
   def display_name
-    "#{first_name} #{last_name}"
+    "#{first_name.capitalize} #{last_name.capitalize}"
```

Comment:
This is potentially one we put out to the team if we haven't already - there are instances where surnames aren't capitalised french stuff like e.g. de Soya and the only [context of when it should be capitalised is the start of a sentence](https://english.stackexchange.com/questions/246157/should-i-capitalize-a-persons-last-name-if-their-name-starts-with-a-lower-case) and in this context, it's not a sentence just displaying the name. Since the supervisors now have control over the names, and not us I think it's ok to leave that up to them to keep tidy (in instances where it's not capitalised in the db but it should be)

Reply:
Yeah raise it to the team, I'm not convinced it's a huge deal